packages/flutter_tools/lib/src/build_runner/resident_web_runner.dart

```diff
@@ -741,7 +743,7 @@ class _DwdsResidentWebRunner extends ResidentWebRunner {
       }
     }
     // Allows browser refresh hot restart on non-debug builds.
-    if (device is ChromeDevice && debuggingOptions.browserLaunch) {
+    if (device is ChromeDevice) {
```

Comment:
This should only be hit if we're running release/profile on chrome

Reply:
Is `device is ChromeDevice && !isRunningDebug` correct? Should it also be `device.device is ChromeDevice` now?